Source/Modules/ModuleRender.h

```diff
@@ -26,6 +26,14 @@ class ModuleRender : public Module
 {
 public:
 
+	enum ModeRender{
+		POSITION = 1,
+		NORMAL = 2,
+		DIFFUSE = 3,
+		SPECULAR = 4,
+		LENGTH = 5
```

Comment:
I'd change this to
```suggestion
		LENGTH
```
Just so, if we were to add more modes, we wouldn't need to change it. Similarly, I think you only need to add the first value of the enum, and the rest will be automatically assigned (so, you only need to do `POSITION = X`, and only when X != 0)